### ISSUE TITLE
The container-cmd field prevents container app to start properly in v3.0.0 (workaround provided)

### ISSUE THREAD

**Issue by seergidev** [openanalytics/shinyproxy#406]
The `container-cmd` field in `application.yml` file does not work well in ShinyProxy 3.0.0 while it was working fine in 2.6.1.

As example, consider this app Dockerfile:
```
FROM eddelbuettel/r2u:22.04

RUN apt-get -y update && apt-get -y upgrade && \
    apt-get -y install --no-install-recommends pandoc && \
    rm -rf /var/lib/apt/lists/* && \
    apt-get clean

RUN install.r rmarkdown flexdashboard shiny ggplot2 dplyr DT 

RUN useradd -m app 
WORKDIR /home/app
COPY myapp /home/app
RUN chown -R app:app /home/app
USER app

EXPOSE 3838

CMD ["R", "-e", "rmarkdown::run('/home/app/index.Rmd',shiny_args=list(port=3838,host='0.0.0.0'))"]
```
and this app specs in `application.yml` file:
```
- id: myapp
  display-name: My App
  description: My App with rmarkdown, shiny and flexdashboard
  container-cmd: ["R", "-e", "rmarkdown::run('/home/app/index.Rmd',shiny_args=list(port=3838,host='0.0.0.0'))"]
  container-image: myapp_image
  container-network: sp-net
  access-groups: [admins, researchers]
```
The above setup is working fine in version 2.6.1, while it fails to start the app container with version 3.0.0.

As workaround, just remove the `container-cmd` line (or comment it) and it will work again:
```
- id: myapp
  display-name: My App
  description: My App with rmarkdown, shiny and flexdashboard
  # container-cmd: ["R", "-e", "rmarkdown::run('/home/app/index.Rmd',shiny_args=list(port=3838,host='0.0.0.0'))"]
  container-image: myapp_image
  container-network: sp-net
  access-groups: [admins, researchers]
```

Because the CMD command is already defined in the Dockerfile, it is not necessary to declare it again in `application.yml` file.

I do not know if it is a bug or it is working as expected now, but since It was working fine in the last version and took me a while to discover the reason why my apps weren't starting after the update to ShinyProxy 3.0.0, I decided to report it.

Thanks.

**Comment by essemenov** [comment]
Whilte it's a bug I use next workaround: adding _shiny.host_  and _shiny.port_ options
`
    container-cmd:
      - "R"
      - "-e"
      - 'options(shiny.host = "0.0.0.0");options(shiny.port = 3838);shiny::runApp("/opt/myapp")'
`

**Comment by LEDfan** [comment]
Hi, this is indeed a bug in 3.0.0. We are in the progress of releasing a new patch release. We waited a few days to see whether new bugs would arise.

In the meantime you can use the docker image `openanalytics/shinyproxy-snapshot:3.0.1-SNAPSHOT-20230303.105513`

**Comment by LEDfan** [comment]
I just released ShinyProxy 3.0.1 that fixes this, therefore I'm closing this issue. Thanks for reporting!

**Comment by seergidev** [comment]
Thanks so much. Incredibly fast fix! Very good piece of software you have here.
